changelog.md

```diff
@@ -18,6 +18,8 @@
 - smarthome.py driver accepts a second port for TLS communication. Port & protocol will be switched according to host protocol (http / https) 
```

Comment:
smarthome.py or SmartHomeNG driver?

Reply:
Both is correct. The drivers filename is still „io_smarthome.py.js” which I don't want  to change.  BTW: the filename of shNG itself is „smarthome.py“, too ;)  The driver will  not work with older versions which require the websocket protocol v3 since this has been deprecated and removed.